scripts/install/get_tendermint.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#set -x
+
+# an examplary download-url
+# https://github.com/tendermint/tendermint/releases/download/v0.34.13/tendermint_0.34.13_linux_amd64.tar.gz
+export TM_MAJORMINOR="0.34"
+export TM_PATCH="13"
+export TM_REPO="https://github.com/tendermint/tendermint"
+
+export TM_VERSION="${TM_MAJORMINOR}.${TM_PATCH}"
+
+export TARGET_PATH="/usr/local/bin"
+export TMP_PATH="/tmp"
+
+error_exit()
+{
+    echo "Error: $1"
+    exit 1
+}
+
+# check for existence
+export TM_EXISTS_VER=$(tendermint version)
+
+if [[ $TM_EXISTS_VER =~ "${TM_MAJORMINOR}" ]]; then
+  echo "tendermint already exists in your current PATH with a sufficient version = $TM_EXISTS_VER"
+  echo "tendermint is located at = $(which tendermint)"
+  exit
+fi
+
+read -r SYSTEM MACHINE <<< $(uname -s -m)
+
+export ARCH="amd64"
+if [[ $MACHINE -eq "aarch64" ]]; then
+  ARCH="arm64"
+fi
+
+if [[ $MACHINE -eq "x86_64" ]]; then
+  ARCH="amd64"
+fi
+
+export RELEASE_URL="${TM_REPO}/releases/download/v${TM_VERSION}/tendermint_${TM_VERSION}_${SYSTEM,,}_${ARCH}.tar.gz"
```

Comment:
I'm getting 
`scripts/install/get_tendermint.sh: line 42: ${TM_REPO}/releases/download/v${TM_VERSION}/tendermint_${TM_VERSION}_${SYSTEM,,}_${ARCH}.tar.gz: bad substitution` here

Reply:
hello tomas,
 
hmmm ... interesting: what version of bash do you use on your workstation?
so whats the output of `/bin/bash --version`?

additionally: what are the system & architecture strings?
what does `uname -s -m` say?

br,
joachim

Reply:
hi Joachim, I'm on mac. The first outputs:
```
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin19)
Copyright (C) 2007 Free Software Foundation, Inc.
```

and the second `Darwin x86_64`

Reply:
hello tomas,
 
ok, thank you for the information - i'll look into it :)

br,
joachim